content/en/docs/cluster-administration/cluster-wide-alerting-and-notification/alerting-policy.md

```diff
@@ -52,19 +48,24 @@ KubeSphere also has built-in policies which will trigger alerts if conditions de
 
 ## Edit an Alerting Policy
 
-To edit an alerting policy after it is created, on the **Alerting Policies** page, click <img src="/images/docs/cluster-administration/cluster-wide-alerting-and-notification/alerting-policies-node-level/edit-policy.png" height="25px"> on the right.
-
-1. Click **Edit** from the drop-down menu and edit the alerting policy following the same steps as you create it. Click **Update** on the **Notification Settings** page to save it.
+To edit an alerting policy after it is created, on the **Alerting Policies** page, click <img src="/images/docs/cluster-administration/cluster-wide-alerting-and-notification/alerting-policies-node-level/edit-policy.png" height="25px"> on the right of the alerting policy.
 
-   ![click-edit](/images/docs/cluster-administration/cluster-wide-alerting-and-notification/alerting-policies-node-level/click-edit.png)
+1. Click **Edit** from the drop-down list and edit the alerting policy following the same steps as you create it. Click **Update** on the **Notification Settings** page to save it.
 
-2. Click **Delete** from the drop-down menu to delete an alerting policy.
+2. Click **Delete** from the drop-down list to delete an alerting policy.
 
 ## View an Alerting Policy
 
-Click an alerting policy on the **Alerting Policies** page to see its detail information, including alerting rules and alerting messages. You can also see the rule expression which is based on the template you use when creating the alerting policy.
+Click the name of an alerting policy on the **Alerting Policies** page to see its detail information, including alerting rules and alerting messages. You can also see the rule expression which is based on the template you use when creating the alerting policy.
 
 Under **Monitoring**, the **Alert Monitoring** chart shows the actual usage or amount of resources over time. **Notification Settings** displays the customized message you set in notifications.
 
-![alert-detail](/images/docs/cluster-administration/cluster-wide-alerting-and-notification/alerting-policies-node-level/alert-detail.png)
+![alerting-policy-details-page](/images/docs/cluster-administration/cluster-wide-alerting-and-notification/alerting-policies-node-level/alerting-policy-details-page.png)
+
+{{< notice note >}}
+
+You can click <img src="/images/docs/cluster-administration/cluster-wide-alerting-and-notification/alerting-policies-node-level/drop-down-list.png" width='20' /> on the upper-right corner to select or custom a time range for the alert monitoring chart.
```

Comment:
`on` -> `in`

Reply:
Changed. I did not notice this before. Thanks very much.